web/docs/project/env-vars.md

```diff
@@ -92,40 +93,44 @@ These files should not be committed to version control, and they are already ign
-
-```shell
-SOME_VAR_NAME=SOMEVALUE wasp start
-```
+You can set environment variables in the `.profile` or a similar file, which will set them permanently, or you can set them temporarily by defining them at the start of a command (`SOME_VAR_NAME=SOMEVALUE wasp start`).
 
- This is not specific to Wasp and is simply how environment variables can be set in the shell.
+This is not specific to Wasp and is simply how environment variables can be set in the shell.
 
-Defining environment variables in this way can be cumbersome even for a single project and even more challenging to manage if you have multiple Wasp projects. Therefore, we do not recommend this as a default method for providing environment variables to Wasp projects. However, it can be useful for occasionally **overriding** specific environment variables because environment variables set this way **take precedence over those defined in `.env` files**.
+Defining environment variables in this way can be cumbersome even for a single project and even more challenging to manage if you have multiple Wasp projects. Therefore, we do not recommend this as a default method for providing environment variables to Wasp projects during development, you should use .env files instead. However, it can be useful for occasionally **overriding** specific environment variables because environment variables set this way **take precedence over those defined in `.env` files**.
 
 ## Defining Env Vars in Production
 
-While in development, we had the option of using `.env` files which made it easy to define and manage env vars. However, in production, we need to provide env vars differently.
+While in development, we had the option of using `.env` files which made it easy to define and manage env vars. However, in production, when our code is built and deployed, we need to provide env vars differently.
 
 ![Env vars usage in development and production](/img/env/prod_dev_fade_2.svg)
 
 ### Client Env Vars
 
-Client env vars are embedded into the client code during the build and shipping process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys).
+Client env vars are embedded into the client code during the build process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys).
 
-You should provide them to the build command, for example:
+You should provide them to the build command that turns client code into static files:
 ```shell
-REACT_APP_SOME_VAR_NAME=somevalue npm run build
+REACT_APP_SOME_VAR_NAME=somevalue REACT_APP_SOME_OTHER_VAR_NAME=someothervalue npm run build
 ```
 
+Notice that the exact moment when you should provide these env vars is during the `npm run build` command that you execute from `.wasp/build/web-app/` after you have ran `wasp build`. Check [deployment docs](../advanced/deployment/manually#3-deploying-the-web-client-frontend) for more details.
```

Comment:
I find the first sentence a little too long (many filler words). Can you simplify it somehow? 

Also, the deployment docs are missing a "the." I see that Grammarly is bad at catching missing articles before links because of the bracket - something to keep in mind.

Reply:
I dropeed the first part, might not be that important! Can't remember exactly since it was some time since I wrote this, but if it will be missing we can add it again. I added missing "the".